model/src/lib.rs

```diff
@@ -26,7 +26,7 @@ mod sqlx_ext; // Must come first because it is a macro.
 
 mod cfd;
 mod contract_setup;
-mod hex_transaction;
+pub mod hex_transaction;
```

Comment:
❓ Is this needed?

Reply:
yes, it's needed, otherwise I can't serialise the message containing `Transaction`...